src/System.ComponentModel.EventBasedAsync/tests/BackgroundWorkerTests.cs

```diff
@@ -133,10 +133,10 @@ private void DoWorkExpectCancel(object sender, DoWorkEventArgs e)
                 return;
             }
 
-            // we want to wait for cancellation - wiat max 100 * timeoutShort millisec
+            // we want to wait for cancellation - wiat max 100 * TimeoutShort millisec
             for (int i = 0; i < 1000; i++)
```

Comment:
I'm guessing the author intended 1000 to be correct, as in the code, as that would make this timeoutShort seconds total.